{"name": "Попытка входа с невалидными данными: неверный логин", "status": "passed", "steps": [{"name": "Подготовка тестовых данных для сценария: неверный логин", "status": "passed", "start": 1758449612026, "stop": 1758449612026}, {"name": "Отправляем POST-запрос для авторизации курьера", "status": "passed", "start": 1758449612026, "stop": 1758449612493}, {"name": "Проверяем код ответа 404", "status": "passed", "start": 1758449612494, "stop": 1758449612494}, {"name": "Проверяем сообщение об ошибке", "status": "passed", "start": 1758449612494, "stop": 1758449612494}], "parameters": [{"name": "test_case", "value": "'неверный логин'"}, {"name": "credentials", "value": "<function TestCourierLogin.<lambda> at 0x000001F23EA093A0>"}, {"name": "expected_status", "value": "404"}, {"name": "expected_message", "value": "'Учетная запись не найдена'"}], "start": 1758449612026, "stop": 1758449612495, "uuid": "130bd959-162d-4aba-a435-d3de0826d210", "historyId": "40b1223d26604bf2b45670cf64633afe", "testCaseId": "505bba1c9113ea857d47e178537c8295", "fullName": "tests.test_courier_login.TestCourierLogin#test_login_negative_cases", "labels": [{"name": "story", "value": "Неуспешная авторизация"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_courier_login"}, {"name": "subSuite", "value": "TestCourierLogin"}, {"name": "host", "value": "What-Is-Lowe"}, {"name": "thread", "value": "10180-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_courier_login"}], "titlePath": ["tests", "test_courier_login.py", "TestCourierLogin"]}